src/Tagger/Repository/Content.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE RankNTypes #-}
 
-module Tagger.ContentRepository where
+module Tagger.Repository.Content(ContentRepository (..), hoist) where
```

Comment:
why do you think it's better to expose explicitly all the things defined in the module?

Reply:
Being explicit enforces separation of concerns and is helpful when projects grow. Having an explicit export list also helps with keeping the API of the module nice and clean by allowing to hide constructors helpers and other internals that shouldn't be exposed.

I will actually create a ticket to make all lists explicit.